src/.ocamlformat

```diff
@@ -0,0 +1,66 @@
+profile=ocamlformat
+quiet=false
+max-iters=10
+comment-check=true
+wrap-fun-args=true
+wrap-comments=false
```

Comment:
@psteckler I think I actually meant this for #1476, sorry!

Reply:
With `wrap-comments` enabled, you have to use asterisks to preserve formatting, or have a sequence of aligned comments.

Here's an example of a comment from our code, edited for that purpose:
```
      (* 0                                                                                                                                                                                                                                    
       * = l.s * l.x                                                                                                                                                                                                                          
       * + r.s * r.x                                                                                                                                                                                                                          
       * + o.s * o.x                                                                                                                                                                                                                          
       * + m * (l.x * r.x)                                                                                                                                                                                                                    
       * + c                                                                                                                                                                                                                                  
       * =                                                                                                                                                                                                                                    
       * l.s * l.s' * l.x'                                                                                                                                                                                                                    
       * + r.s * r.s' * r.x'                                                                                                                                                                                                                  
       * + o.s * o.s' * o.x'                                                                                                                                                                                                                  
       * + m * (l.s' * l.x' * r.s' * r.x')                                                                                                                                                                                                    
       * + c                                                                                                                                                                                                                                  
       * =                                                                                                                                                                                                                                    
       * (l.s * l.s') * l.x'                                                                                                                                                                                                                  
       * + (r.s * r.s') * r.x'                                                                                                                                                                                                                
       * + (o.s * o.s') * o.x'                                                                                                                                                                                                                
       * + (m * l.s' * r.s') * l.x' r.x'                                                                                                                                                                                                      
       * + c                                                                                                                                                                                                                                  
      *)
```
The `*`s look funny here, because they pun multiplication.